app/storage/metadata_parser.py

```diff
@@ -88,6 +88,10 @@ def _validate_metadata_is_present(metadata, required_metadata):
     :param required_metadata:
     """
     for metadata_field in required_metadata:
```

Comment:
technically this is not correctly named now as its no longer a list of `required_metadata`